fix carpentry >> carpentries

maneesha opened the pull request.

elletjies commented:
@OscarSiba, for some reason, I can't see the preview through the deploy but can here: https://github.dev/carpentries/carpentries.org/pull/491/files. Please feel free to change the release date if you wish.

OscarSiba commented:
I also can't see the preview... Any idea why @maneesha? 
?

maneesha commented:
@OscarSiba and @elletjies What do you see when you click on the "😎 Deploy Preview" link?

maneesha commented:
OK, I see now that the site preview builds but you were not able to see this blog post.  This could be one of a few reasons. I have updated this repo's README to make the instructions clearer:

1.  The keys in the frontmatter of the file need to be in lowercase.  Here we had `Date` instead of `date`
2. Strings in the frontmatter need to be in straight quotes  (`"Name"` and not `“Name”`)
3. The file name should not include the date and should include alphanumeric and underscore characters only (no spaces, punctuation, or other special characters)
4. The file name should have the `.md` extension

I have updated the README instructions to reflect all of the above.  I have also made all the updates to this PR and the preview now works.  

(edit-typo)

elletjies commented:
Thank you for this @maneesha. The .md is my fault :( @OscarSiba I have also updated the Blog post templates.

maneesha commented:
@OscarSiba is this still awaiting your review?

OscarSiba commented:
Yes it is @maneesha. Thinking of bringing the date forward, but awaiting to see if another blog post we are anticipating will come in. Please hold on.

elletjies commented:
@froggleston could you merge this please?

froggleston commented:
This now renders better after some CSS changes, but on smaller screens, the fixed width table gets shifted underneath the sidebar so looks weird:

![{65132099-998C-4D3A-B8D4-3738EB7FD202}](https://github.com/user-attachments/assets/4f6c93e2-f1ee-467f-aaa3-1ec90e35e4e1)

I'll see if I can adjust this in a future PR.
